tools/ui-generator/presto-sample/README.md

```diff
@@ -0,0 +1,35 @@
+# install
```

Comment:
Is this relevant? Can we stick to only webapp for now.

Reply:
Yup, that's carried over from DUI README